packages/dialtone-vue2/.storybook/preview-head.html

```diff
@@ -2,4 +2,8 @@
   .sbdocs.sbdocs-content {
     max-width: 2000px;
   }
+
+  * {
+    box-sizing: border-box;
+  }
```

Comment:
Noticed we didn't have box-sizing: border-box set globally in storybook, causing inconsistencies with spacing between storybook and product.